src/onyx/schema.clj

```diff
@@ -185,12 +195,21 @@
    (s/optional-key :onyx.peer/retry-start-interval) s/Int
    (s/optional-key :onyx.peer/join-failure-back-off) s/Int
    (s/optional-key :onyx.peer/drained-back-off) s/Int
-   (s/optional-key :onyx.peer/peer-not-ready-back-off) s/Int
    (s/optional-key :onyx.peer/job-not-ready-back-off) s/Int
+   (s/optional-key :onyx.peer/peer-not-ready-back-off) s/Int
    (s/optional-key :onyx.peer/fn-params) s/Any
    (s/optional-key :onyx.peer/backpressure-check-interval) s/Int
    (s/optional-key :onyx.peer/backpressure-low-water-pct) s/Int
    (s/optional-key :onyx.peer/backpressure-high-water-pct) s/Int
+   (s/optional-key :onyx.peer/state-log-impl) StateLogImpl
+   (s/optional-key :onyx.peer/state-filter-impl) StateFilterImpl
+   (s/optional-key :onyx.bookkeeper/starting-port) s/Int
```

Comment:
When you use it in local-quorum? mode, it'll spin up three instances. Generally users will only use one instance per node, so I like port, in general, but maybe we should have an extra config option that specifies which ports to use for a local quorum?